packages/nextcloud/lib/src/api/provisioning_api/provisioning_api.openapi.dart

```diff
@@ -6232,11 +6232,11 @@ typedef UserDetailsQuota_Quota = ({num? $num, String? string});
 
 @BuiltValue(instantiable: false)
 sealed class $UserDetailsQuotaInterface {
-  num get free;
+  num? get free;
```

Comment:
I disagree with this change.
```json
            "UserDetailsQuota": {
                "type": "object",
                "properties": {
                    "free": {
                        "anyOf": [
                            {
                                "type": "number",
                                "format": "double"
                            },
                            {
                                "type": "integer",
                                "format": "int64"
                            }
                        ]
                    },
                    ...
                  },
                },
```

I don't see where null is allowed here.
The value has to be either a _non-nullable_ `integer` or a _non-nullable_ `double`. The union of both in dart is a _non-nullable_ `num`.

IMO the upstream specs are broken.

Reply:
Ah yes, I was probably thinking again that anyOf means >=0 but it means >=1 matching schemas.
I'll check if this is also something that is wrong in the openapi-extractor.